test/integration/noderesourcetopology_test.go

```diff
@@ -553,6 +553,32 @@ func TestTopologyMatchPlugin(t *testing.T) {
 			},
 			expectedNodes: []string{"fake-node-1"},
 		},
+		{
+			name: "Scheduling Best-Effort pod with least-allocatable strategy scheduler",
+			pods: []*v1.Pod{
+				st.MakePod().Namespace(ns.Name).Name("topology-aware-scheduler-pod").SchedulerName(leastAllocatableScheduler).Container(pause).Obj(),
+			},
+			// in this test, both nodes are expected to get the maximum score.
+			// in case of multiple nodes with the same highest score the selection done randomly.
+			// thus, we can't tell exactly which node will get selected, so the only option is to accept both of them,
+			// and it's still good enough for making sure that pod not stuck in pending state and/or the program isn't crashing
+			// same goes to the other balanced-allocation and most-allocatable strategy tests below
```

Comment:
minor: this is a good comment, but it applies to all the three cases (leastAllocatable, mostAllocatable, balancedAllocation), right? moving it between lines 555 and 556 and explicitely mention the three strategies can be clearer.

Reply:
sure. done